test_runner/regress/test_disk_usage_eviction.py

```diff
@@ -91,9 +95,38 @@ def warm_up_tenant(self, tenant_id: TenantId):
         with self.neon_env.postgres.create_start("main", tenant_id=tenant_id, lsn=lsn) as pg:
             self.pg_bin.run(["pgbench", "-S", pg.connstr()])
 
+    def pageserver_start_with_disk_usage_eviction(
+        self, period, max_usage_pct, min_avail_bytes, mock_behavior
+    ):
+        magic = str(uuid.uuid4())
+
+        disk_usage_config = {
+            "period": period,
+            "max_usage_pct": max_usage_pct,
+            "min_avail_bytes": min_avail_bytes,
+            "mock_statvfs": {
+                "behavior": mock_behavior,
+                "magic": magic,
+            },
+        }
+
+        enc = toml.TomlEncoder()
+
+        self.neon_env.pageserver.start(
+            overrides=(
+                "--pageserver-config-override=disk_usage_based_eviction="
+                + enc.dump_inline_table(disk_usage_config).replace("\n", " "),
+            ),
+        )
+
+        def statvfs_called():
+            assert self.neon_env.pageserver.log_contains(".*running mocked statvfs.*" + magic)
```

Comment:
This 'magic' feels like overkill, I think checking for the string "running mocked statvfs" is enough